src/unix/kqueue.c

```diff
@@ -82,7 +82,8 @@ int uv__io_fork(uv_loop_t* loop) {
        process. So we sidestep the issue by pretending like we never
        started it in the first place.
     */
-    uv__store_relaxed(&uv__has_forked_with_cfrunloop, 1);
+    atomic_store_explicit(
+        &uv__has_forked_with_cfrunloop, 1, memory_order_relaxed);
```

Comment:
It's kind of nit-picky but style (hope I get the indentation right):
```suggestion
    atomic_store_explicit(&uv__has_forked_with_cfrunloop,
                          1,
                          memory_order_relaxed);
```
Ditto below 2x.